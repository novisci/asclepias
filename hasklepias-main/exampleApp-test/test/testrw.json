{"example":[{"totalProcessed":2,"attritionInfo":[{"attritionLevel":{"tag":"SubjectHasNoIndex"},"attritionCount":0},{"attritionLevel":{"contents":[1,"dummy"],"tag":"ExcludedBy"},"attritionCount":0},{"attritionLevel":{"tag":"Included"},"attritionCount":2}]},{"contents":{"cohortData":[[["a",1],[5,true]],[["b",1],[5,true]]],"attributes":[{"name":"myVar1","type":"Count","attrs":{"getDerivation":"","getShortLabel":"somelabel","getPurpose":{"getRole":["Outcome"],"getTags":[]},"getLongLabel":"another label"}},{"name":"myVar2","type":"Bool","attrs":{"getDerivation":"","getShortLabel":"","getPurpose":{"getRole":[],"getTags":[]},"getLongLabel":""}}]},"tag":"RW"}]}