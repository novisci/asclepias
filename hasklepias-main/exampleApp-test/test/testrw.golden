{"example":[{"attritionInfo":[{"attritionCount":0,"attritionLevel":{"tag":"SubjectHasNoIndex"}},{"attritionCount":0,"attritionLevel":{"contents":[1,"dummy"],"tag":"ExcludedBy"}},{"attritionCount":2,"attritionLevel":{"tag":"Included"}}],"totalProcessed":2},{"contents":{"attributes":[{"attrs":{"getDerivation":"","getLongLabel":"another label","getPurpose":{"getRole":["Outcome"],"getTags":[]},"getShortLabel":"somelabel"},"name":"myVar1","type":"Count"},{"attrs":{"getDerivation":"","getLongLabel":"","getPurpose":{"getRole":[],"getTags":[]},"getShortLabel":""},"name":"myVar2","type":"Bool"}],"cohortData":[[["a",1],[5,true]],[["b",1],[5,true]]]},"tag":"RW"}]}